Dockerfile

```diff
@@ -30,3 +30,22 @@ ENV JAVA_HOME /usr/lib/jvm/default-java
 # Set up baselisk.
 RUN wget https://github.com/bazelbuild/bazelisk/releases/download/v1.10.1/bazelisk-linux-amd64 \
     -O /usr/local/bin/bazelisk && chmod a+x /usr/local/bin/bazelisk
+
+# Rust is needed for authorization logic
+ARG rustup_dir=/usr/local/cargo
+ENV RUSTUP_HOME ${rustup_dir}
+ENV CARGO_HOME ${rustup_dir}
+ENV PATH "${rustup_dir}/bin:${PATH}"
+RUN curl --location https://sh.rustup.rs > /tmp/rustup \
+  && sh /tmp/rustup -y --default-toolchain=none \
+  && chmod a+rwx ${rustup_dir} \
+  && rustup --version
+
+## Set the right version of rust
+ARG RUST_VERSION=nightly-2021-06-30
```

Comment:
This is a very old nightly version; would something more recent work?

Reply:
I just confirmed that the latest nightly (November 1) really does not work because some cargo feature breaks ("raw" ?). Gogul and I did a short search for the latest-ish nightly that works when he was getting rust to build with blaze, and this is what worked then :)